Dockerfile

```diff
@@ -355,6 +354,10 @@ RUN pip install bcolz && \
     pip install pexpect && \
     pip install pickleshare && \
     pip install Pillow && \
+    # Install openslide and its python binding
+    apt-get install -y openslide-tools && \
+    # b/152402322 install latest from pip once is in: https://github.com/openslide/openslide-python/pull/76
+    pip install git+git://github.com/rosbo/openslide-python.git@fix-setup && \
```

Comment:
Any reason we're not using one of the kaggle repos for this?

Reply:
This is a GH fork. I could have created the fork under the kaggle org but all the repos there are managed using Terraform and I figured it was overkill for this.